app/controllers/concerns/error_handling.rb

```diff
@@ -27,6 +28,11 @@ def handle_access_denied(exception)
       handle_error(exception, :forbidden)
     end
 
+    def handle_forgery_protection(exception)
+      System::ErrorReporting.report_error(exception)
+      handle_error(exception, :forbidden)
```

Comment:
Forbidden because, as @thomasmaas told me: https://stackoverflow.com/questions/23478370/what-response-should-be-sent-back-a-when-cross-site-request-forgery-csrf-is-de

Reply:
although it's not an accepted answer…I think it makes sense.